ModuleTwoFirstTaskDebug/Task1/Product.cs

```diff
@@ -0,0 +1,54 @@
+﻿// <copyright file="Product.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace ModuleTwoDebugTask
+{
+    /// <summary>
+    /// Some product with it's properties.
+    /// </summary>
+    public class Product
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Product"/> class.
+        /// Sets name and price of the product.
+        /// </summary>
+        /// <param name="name">Name.</param>
+        /// <param name="price">Price.</param>
+        public Product(string name, double price)
+        {
+            this.Name = name;
```

Comment:
`this` keyword is redundant

Reply:
This is one of the style rules of the project. If we don't follow them - making code review takes more effort.

Reply:
Fixed in 9e61849.